Tools-Override/FrameworkTargeting.targets

```diff
@@ -228,11 +228,18 @@
 
   <Target Name="BinPlaceFiles"
           Condition="'@(BinPlaceDir)' != ''"
-          DependsOnTargets="GetBinPlaceItems" >
+          DependsOnTargets="GetBinPlaceItems"
+          Inputs="@(BinPlaceDir);%(BinPlaceDir.ItemName)"
```

Comment:
Also if you want to force it to run, of course removing inputs and outputs achieves that